source/renderer/app/components/widgets/WalletTestEnvironmentLabel.js

```diff
@@ -3,24 +3,40 @@ import { defineMessages, intlShape } from 'react-intl';
 import styles from './WalletTestEnvironmentLabel.scss';
 
 const messages = defineMessages({
-  testnetLabel: {
+  staging: {
+    id: 'test.environment.stagingLabel',
+    defaultMessage: '!!!Staging vx',
+    description: 'Label for staging network with version.'
+  },
+  testnet: {
     id: 'test.environment.testnetLabel',
     defaultMessage: '!!!Testnet vx',
     description: 'Label for testnet with version.'
   },
+  development: {
+    id: 'test.environment.developmentLabel',
+    defaultMessage: '!!!Development vx',
+    description: 'Label for development with version.'
+  }
 });
 
-export default class WalletTestEnvironmentLabel extends Component {
+type Props = {
+  network: string,
+};
+
+export default class WalletTestEnvironmentLabel extends Component<Props> {
 
   static contextTypes = {
     intl: intlShape.isRequired,
   };
 
   render() {
+    const { network='development' } = this.props;
```

Comment:
@daniloprates let's not use a default value here - we should move it into this file:
https://github.com/input-output-hk/daedalus/blob/devops-814-network-testnet/source/common/environment.js#L15
...so that it looks something like this:
```
NETWORK: process.env.NETWORK || 'development',
```